room1: [-6.13, 8.2, 0.0]
room2: [-5.97, -8.21, 0.0]
hall: [0.0, 0.0, 0.0]
office1: [-5.37, 1.13, 0.0]
office2: [-5.22, -1.37, 0.0]
storage1: [12.5, 7.89, 0.0]
storage2: [12.47, -7.72, 0.0]